### PR TITLE
Add SD-JWT Reference Implementation proposal

### PR DESCRIPTION
**Pull request by danielfett**

### CODE REVIEW

File: projects/sd-jwt-python.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+This library implements the Selective Disclosure for JWTs specification, which is an important building block for future digital identity wallets.
+
+# Current Code of Conduct
+
+none
+
+# TAC Sponsor
+
+--
```
Comment (tkuhrt) [inline projects/sd-jwt-python.md:21]: Happy to serve as a sponsor.

```suggestion
- [Tracy Kuhrt](https://github.com/tkuhrt)
```

**Review by tkuhrt — APPROVED** [review]
Thank you for the submission. I am happy to serve as a sponsor for this. I know that you have also had some conversations with @hkny who is also working on a Python implementation of SD-JWT. Not sure of how those conversations have progressed or whether you would consider him to be an initial maintainer.

A couple of points:

1. Before we bring in the source base to the OWF, one of the requirements is that the code has [DCO sign off](https://developercertificate.org/). Please let me know if you need instructions on doing this.
2. What are your thoughts about making this a generic SD-JWT lab and working with the maintainers of #8 proposal? We can continue to have separate repos for the different languages.

**Comment by hkny** [comment]
Hi @danielfett,
I'd like to be added as one of the initial maintainers if that's okay with you. Also, let's find out if we can merge with https://github.com/openwallet-foundation/project-proposals/pull/8 as mentioned during our call today.
BR

**Review by skounis — APPROVED** [review]

**Comment by tkuhrt** [comment]
Hi, @danielfett. We will be reviewing this pull request in this week's TAC meeting on Wednesday May 17 at 7:00 AM US/Pacific. It would be great if you can join us. Please see the [OpenWallet Foundation calendar](https://calendar.google.com/calendar/u/0?cid=Y181MjdhMWRiNjIxZmU3MDczMzg0MjFhOWM0YzdlZjJmNjE2MGI5NzM4MzhjZmRiYjUwZjBkYjMyYWQwZmQyMGViQGdyb3VwLmNhbGVuZGFyLmdvb2dsZS5jb20) for details on the meeting.

**Comment by tkuhrt** [comment]
**Approved by TAC at the [May 17, 2023 meeting](https://openwallet-foundation.github.io/tac/meetings/2023-05-03/)**

@danielfett : To bring this into the [openwallet-foundation-labs organization](https://github.com/openwallet-foundation-labs), we need to complete the following items:

- [ ] Add [DCO sign off](https://developercertificate.org/) to https://github.com/danielfett/sd-jwt. Please let me know if you need instructions on doing this.
- [ ] Once the above is done, transfer the repository to me (@tkuhrt)
- [ ] I will then transfer the repo to the openwallet-foundation-labs organization and create a team for the maintainers
- [ ] Merge this PR

**Comment by tkuhrt** [comment]
Repository created: https://github.com/openwallet-foundation-labs/sd-jwt-python

**Pull request merged**